include/SFML/Graphics/Sprite.hpp

```diff
@@ -108,7 +108,7 @@ class SFML_GRAPHICS_API Sprite : public Drawable, public Transformable
     /// \brief Disallow setting from a temporary texture
     ///
     ////////////////////////////////////////////////////////////
-    void setTexture(Texture&& texture, bool resetRect = false) = delete;
+    void setTexture(Texture&& texture, bool resetRect) = delete;
```

Comment:
This is going against the purpose of this declaration. Calling setTexture with an rvalue reference but no boolean argument will not match this signature, so the setTexture taking const reference will be used instead, missing the opportunity to show an error.

Reply:
Good catch, I didn't think that one through! Fixing...